### PR TITLE
Updates To Desktop & Mobile Typography

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/127030

Updates to the design system: https://paper.dropbox.com/doc/Views-Design-System-Change-Log-MbGty1uf14NH9sfKIl1zE

### CODE REVIEW

File: src/resources/postcss/base/typography/_headings.pcss [hunk 1]
```diff
@@ -85,13 +85,37 @@
...
 		color: var(--color-text-primary);
 		font-family: var(--font-family-sans-serif);
-		font-size: var(--font-size-1);
+		font-size: var(--font-size-3);
 		font-weight: var(--font-weight-bold);
-		line-height: var(--line-height-2);
+		line-height: var(--line-height-3);
 
 		@media (--viewport-medium) {
-			font-size: var(--font-size-2);
```
Comment (paulmskim) [inline src/resources/postcss/base/typography/_headings.pcss]: should just remove this as it's the same as mobile

File: src/resources/postcss/base/typography/_headings.pcss [hunk 2]
```diff
@@ -61,9 +61,9 @@
 	.tribe-common-h4 {
 		color: var(--color-text-primary);
 		font-family: var(--font-family-sans-serif);
-		font-size: var(--font-size-5);
+		font-size: var(--font-size-7);
```
Comment (paulmskim) [inline src/resources/postcss/base/typography/_headings.pcss]: we actually need to keep this at `--font-size-5` and `--line-height-1` for mobile, but below, add a media query for `--viewport-medium` with the desktop size (`--font-size-7`)

File: src/resources/postcss/base/typography/_headings.pcss [hunk 3]
```diff
@@ -61,9 +61,9 @@
...
 		font-weight: var(--font-weight-bold);
-		line-height: var(--line-height-2);
+		line-height: var(--line-height-1);
 	}
```
Comment (paulmskim) [inline src/resources/postcss/base/typography/_headings.pcss]: let's also add `.tribe-common-h4--alt` with `font-weight` of `regular`, much like `.tribe-common-h3--alt`.

**Comment by juanfra** [comment]
Thanks Paul! All set

**Pull request merged**